keps/sig-node/2400-node-swap/README.md

```diff
@@ -290,6 +309,10 @@ nodes could improve better resource pressure handling and recovery.
 
 This user story is addressed by scenario 1 and 2, and could benefit from 3.
 
+Note: critical / high-priority pods would not be able to access swap, but can
```

Comment:
Is this true? A critical/ high priority pod that is Burstable would still swap out?

Reply:
It will be soon, after https://github.com/kubernetes/kubernetes/pull/125277 is merged.
This topic was already brought up to sig-node meeting and there was a consensus that this path forward makes sense.

As written in this KEP update, customizability will be enhanced with follow-up KEPs (that are WIP).